### DISCUSSION TITLE
web button timer configuration- button instead switch

### DISCUSSION THREAD

**Discussion by to6koto6ko** [arendst/Tasmota#18891] [Q&A] [answered]
Hello there,
I am new to Tasmota, but I think it is close to becoming my new passion.
I have been trying for more than 3 days to configure my Tasmota Relay Webbutton to act as a "button" instead of "toggle".
The setup is - One esp01s (tasmotized) with relay board v4. It is configured as a Relay_i 1. 
Perhaps I don`t understand the logic behind it but I am still unable to reach the result.
I need to configure the relay as button to open my garage door and every time when I push the web button - the circuit to be switched for ~1 sec.
With web api the result is perfect, but I want to configure the button.
My webapi link: 
http://xx.xx.xx.xx/cm?&cmnd=Backlog%20Power%20Toggle%3BPower1%20off
Any help will be highly appreciated. Not only to solve my problem, but to figure out the logic behind the Tasmota.
Best regards,
Todor

**Comment by sfromis** [comment, marked as answer]
Acting as a "Button" **is** a toggle, no way of "instead" with that config. If what you want is for the web button to act as a locking Switch (instead of a Button), that is not a feature of Tasmota, as a web button can only provide a momentary "click" when pushed. If you want a one-second pulse, that is not available for the web button as such, but you can configure `PulseTime` for the relay, to have it automatically turn off again, after being turned on.

**Reply by to6koto6ko** [reply]
Thank you very much - this is exactly what I wanted! 
The command is PulseTime 10
I can now explain to myself what is exactly the logic behind Tasmota in this case.
Thank you one more time - it works! I spend a lot of time to reading the manuals...
